package.json

```diff
@@ -34,6 +34,7 @@
     "backbone.marionette@npm:3.5.1/underscore": "^1.13.6"
   },
   "devDependencies": {
+    "@preact/preset-vite": "^2.9.4",
```

Comment:
Why do we need it there? Don't we need to move the dependency to the related package?